pkgs/development/python-modules/tempita/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, fetchFromGitHub, nose }:
+
+buildPythonPackage rec {
+  version = "0.5.3-2016-09-28";
+  name = "tempita-${version}";
+
+  src = fetchFromGitHub {
+    owner = "gjhiggins";
+    repo = "tempita";
+    rev = "47414a7c6e46a9a9afe78f0bce2ea299fa84d10";
+    sha256 = "0f33jjjs5rvp7ar2j6ggyfykcrsrn04jaqcq71qfvycf6b7nw3rn";
+  };
+
+  buildInputs = [ nose ];
+
+  meta = {
+    homepage = https://github.com/gjhiggins/tempita;
+    description = "A very small text templating language";
```

Comment:
Would be nice to have a license here (ref https://github.com/NixOS/nixpkgs/issues/43716), but the repo doesn't have any and we would have to assume worst-case and make it unfree, but that would be kinda disappointing to have all its dependents be unfree too because of that, so I'd be okay with leaving it out.

Reply:
According the original author's archived website (https://web.archive.org/web/20150301124031/http://pythonpaste.org:80/tempita/license.html), it is MIT licensed.